src/data_availability.rs

```diff
@@ -600,6 +518,43 @@ impl DataAvailability {
 
         Ok(())
     }
+
+    async fn ask_for_catchup_blocks(
+        &mut self,
+        ip: String,
+        sender: tokio::sync::mpsc::Sender<SignedBlock>,
+    ) -> Result<(), Error> {
+        info!("📡 Streaming data from {ip}");
+        let Ok(mut stream) = RawDAListener::new(&ip, crate::model::BlockHeight(0)).await else {
```

Comment:
are you sure you allways want to catchup from block 0 there ? I think we should give the last known block height as we do in DAListener::build ?

Reply:
Ah yes, didn't have that info in the consensus but I do now